src/express.scm

```diff
@@ -72,11 +76,13 @@
 	(LambdaLink
 		(VariableList (VariableNode "$emo") (VariableNode "$expr"))
 		(RandomNumberLink
-			(GetLink (VariableNode "$int-min")
+			(GetLink
+				(TypedVariable (Variable "$int-min") (Type "NumberNode"))
 				(StateLink (ListLink
```

Comment:
As above. There is nothing particularly wrong with this change, but if this is making your code run differently, then there is a bug somewhere, anyway.  This just moves the bug around or hides the bug.